chart/values.yaml

```diff
@@ -15,13 +15,19 @@ jobNodeSelector: ""
 
 resources:
   limits:
-    cpu: "1000m"
-    memory: "2Gi"
+    cpu: "500m"
+    memory: "1Gi"
   requests:
-    cpu: "1000m"
-    memory: "2Gi"
+    cpu: "500m"
+    memory: "1Gi"
```

Comment:
can we make it even smaller?

Reply:
Yes, this is for pod where run `terraform apply` which doesn't require much resources